server/src/shared/app/base-controller.ts

```diff
@@ -14,6 +15,11 @@ export abstract class BaseController {
     }
   }
 
+  public redirect<Res extends express.Response>(res: Res, url: string, params: ParamList) {
```

Comment:
nice, I'm gonna yoink that for the Waterpark BaseController too 😄

Reply:
time for the loo labs framework to be pulled out ;)